template.yaml

```diff
@@ -718,7 +719,7 @@ Resources:
         - Key: IncludedInBackup
           Value: 'true'
 
-  NvaPersonInfoTable:
+  NvaIdTypeObjectsTable:
```

Comment:
heads up, this can delete table, but I guess its ok

Reply:
or leave garbage, then we need to manually tidy in every env

Reply:
Table is new and unused, but thanks for warning me!